tests/backup/backup_share_test.go

```diff
@@ -4123,11 +4123,15 @@ var _ = Describe("{SwapShareBackup}", func() {
 			ctx, err := backup.GetNonAdminCtx(users[i], commonPassword)
 			log.FailOnError(err, "Fetching non admin ctx")
 			_, err = DeleteBackup(backupName, backupUIDList[i], orgID, ctx)
-			dash.VerifySafely(err, nil, fmt.Sprintf("Verifying backup deletion - %s", backupName))
-			backupDriver := Inst().Backup
-			backupEnumerateReq := &api.BackupEnumerateRequest{
-				OrgId: orgID,
-			}
+			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting backup - %s of user %s", backupName, users[i]))
+		}
+		backupDriver := Inst().Backup
+		backupEnumerateReq := &api.BackupEnumerateRequest{
+			OrgId: orgID,
+		}
+		for i := numberOfUsers - 1; i >= 0; i-- {
+			ctx, err := backup.GetNonAdminCtx(users[i], commonPassword)
```

Comment:
Is it possible to make use of DeleteBackupAndWait?

Reply:
done

Reply:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/3887/